---
https://patchwork.kernel.org/patch/10281327/
In latest e2fsprogs (1.44.0) definition of ext2_ext_attr_entry has
removed member e_value_block, as currently ext* doesn't support it set
anyway.

So remove such check so that we can pass compile.

diff --git a/btrfs-convert.c b/btrfs-convert.c
index 02e5cdb..2a82a74 100644
--- a/btrfs-convert.c
+++ b/btrfs-convert.c
@@ -712,8 +712,7 @@ static int ext2_xattr_check_entry(struct ext2_ext_attr_entry *entry,
 {
 	size_t value_size = entry->e_value_size;
 
-	if (entry->e_value_block != 0 || value_size > size ||
-	    entry->e_value_offs + value_size > size)
+	if (value_size > size || entry->e_value_offs + value_size > size)
 		return -EIO;
 	return 0;
 }
